nemo/collections/asr/parts/submodules/tdt_loop_labels_computer.py

```diff
@@ -327,7 +334,19 @@ def loop_labels_torch(
                     time_indices=time_indices_current_labels,
                     logits=logits if self.preserve_alignments else None,
                     labels=labels if self.preserve_alignments else None,
-                    confidence=self._get_confidence_tensor(F.log_softmax(logits[:, :-num_durations], dim=-1))
+                    confidence=torch.cat(
```

Comment:
Use `torch.stack` to concat along new dimension (instead of multiple unsqueeze)

Reply:
Thanks, done.